python/lrscheduler.py

```diff
@@ -0,0 +1,150 @@
+# This class serves as a simpler combined interface to apply a learning rate scheduler
+# There are two main different ways to support this, through either
+# tf.keras.callbacks.LearningRateScheduler
+# tf.keras.optimizers.schedules
+# The two methods each has some nice features that are not trivially available in the other
+# note two methods can not be used together, since using schedules takes up the place for learning rate
+
+import numpy as np
+import tensorflow.keras.optimizers.schedules as schedules
+import tensorflow.keras.callbacks as callbacks
+import json
+from callbacks import PrintLearningRate
+
+WARM_UP_EPOCHS = 5
```

Comment:
I'm not sure about this but would it be better to have `WARM_UP_EPOCHS` in the config?